website/source/docs/providers/aws/r/api_gateway_deployment.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "aws"
+page_title: "AWS: aws_api_gateway_deployment"
+sidebar_current: "docs-aws-resource-api-gateway-deployment"
+description: |-
+  Provides an API Gateway Deployment.
+---
+
+# aws\_api\_gateway\_deployment
+
+Provides an API Gateway Deployment.
+
+**Note:** Depends on having `aws_api_gateway_method` inside your rest api. To ensure this
```

Comment:
The `-> ` prefix here would draw the blue box I talked about earlier and make this note more visible.